facia/app/controllers/FaciaController.scala

```diff
@@ -188,6 +189,18 @@ trait FaciaController
     val futureResult = futureFaciaPage.flatMap {
       case Some((faciaPage, _)) if nonHtmlEmail(request) =>
         successful(Cached(CacheTime.RecentlyUpdated)(renderEmail(faciaPage)))
+      case Some((faciaPage: PressedPage, _)) if request.forceDCR && request.isJson =>
```

Comment:
If I've understood correctly, the next case (`case Some((faciaPage: PressedPage, targetedTerritories))`) handles a situation where a front includes targeted territories (i.e. au-specific containers). If a front includes territory specific containers we set a vary header to force the cache to be split on territory for that front.

What's the intention for creating a separate case for forceDCR && isJson? How does the interact with any potential caching?

Reply:
I think we initially moved it because we were having a hard time getting the DotcomFrontsRenderingDataModel to play nice with the `Cached` method!

I think now; there is something nice to be said for having a new `case` for DCR, but perhaps it's better to put it in the case statement with `targetedTerritories` so we keep that existing caching functionality?

Reply:
(now done)

Reply:
Looks good :) Hopefully i wasn't talking rubbish!